docs/V5-Migration-Guide.md

```diff
@@ -23,11 +23,17 @@ If you are upgrading from a version before 4.0.0, please see the [previous migra
 
 ### Breaking Changes:
 
-#### [has consolidation](https://github.com/dojo/framework/pulls)
+#### [`has` module consolidation](https://github.com/dojo/framework/pull/182)
 
 Part of the cleanup started with the 3.0 consolidation of packages into `@dojo/framework`, this release of Dojo moves all the `has` related modules in a single `has` module, `@dojo/framework/has/has`. If you run the `dojo/cli-upgrade-app` command all references to removed has modules should be automatically updated.
 
-#### [Support for `before` middleware in stores](https://github.com/dojo/framework/pulls)
+#### [`classes` property with the `Themed` mixin](https://github.com/dojo/framework/pull/164)
+
+A new API that supports passing classes keyed by a widgets theme key is now available for all widgets that use the `@dojo/framework/widget-core/meta/Themed` mixin.
+
+This is not a traditional breaking changes as it will only affect themed widgets that have a `classes` property specified on its API.
```

Comment:
```suggestion
This is not a traditional breaking change as it will only affect themed widgets that have a `classes` property specified on its API.
```